Form-Controls/index.html

```diff
@@ -13,15 +13,50 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+         <label for="name">Name</label>
+         <input type="text" id="name" name="name" required>
+         <br>
+         <br>
+         <label for="email">Email</label>
+         <input type="text" id="email" name="name" required>
+         <br>
+         <hr>
+         <br>
+         <fieldset>
+          <legend>Choose color of t-shirts</legend>
+
+         <input type="radio" id="Black" name="color" value="Black" required>
+         <label for="Black">Black</label>
+
+         <input type="radio" id="White" name="color" value="White">
+         <label for="White">White</label>
+
+         <input type="radio" id="Yellow" name="color" value="Yellow">
+         <label for="Yellow">Yellow</label>
+         </fieldset>
+         
+         <br>
+         <br>
+         <label for="size">
+          Choose your size
+          </label>
+         <select name="size" id="size" required>
```

Comment:
Right now, the default size of your dropdown is 'XS'. If I forget to select from this dropdown, I'll get a size XS t-shirt. Can you think of a better way to implement your dropdown so that the user is forced to actively select a size?